Automattic/sensei
# Update dummy data

This PR removes:
- Obsolete courses and their associated course categories.
- References to Woo or WooThemes.
- `_course_woocommerce_product` post meta.
- References to purchasing Sensei.

## Testing
1. Import `dummy_data.xml` to a Sensei site that has never had it imported before.
2. Check that 4 courses have been created - _How to Play the Piano_, _How to play the guitar_, _Sensei Theming 101_ and _Create on Online Course_.
3. Check that a `sensei` user was created, but not `wooadmin` or `wooteam` users.
4. Check that _WooCommerce (WooThemes)_ and _WooThemes (WooThemes)_ course categories were not created.
5. Ensure that none of the course categories include _WooThemes_ in their name.
6. Ensure that none of the modules include _WooThemes_ in their name.
7. Take the _Create an Online Course_ course and ensure there are no references to purchasing Sensei.
